src/Microsoft.ApplicationInsights/Extensibility/TelemetryConfiguration.cs

```diff
@@ -234,6 +234,18 @@ public ITelemetryChannel TelemetryChannel
         /// </remarks>
         public IApplicationIdProvider ApplicationIdProvider { get; set; }
 
+        /// <summary>
+        /// Gets a collection of string indicating if an experimental feature should be enabled.
+        /// The presence of a string in this collection will be evaluated as 'true'.
+        /// </summary>
+        /// <remarks>
+        /// This property allows the dev team to ship and evaluate features before adding these to the public API.
+        /// We are not committing to support any features enabled through this property.
+        /// Use this at your own risk.
+        /// </remarks>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public IList<string> ExperimentalFeatures { get; internal set; } = new List<string>(0);
```

Comment:
nit: did you want to set capacity to 0? does it help with perf in this case?

Reply:
For most users, this will never be used.
I think by default a List initializes with size 5 or 6.